app/src/main/java/org/kiwix/kiwixmobile/zim_manager/local_file_transfer/LocalFileTransferActivity.java

```diff
@@ -203,6 +217,7 @@ public void onFilesForTransferAvailable(@NonNull ArrayList<FileItem> filesForTra
   }
 
   private void displayFileTransferProgress(@NonNull ArrayList<FileItem> filesToSend) {
+    textBannerFilesForTransfer.setVisibility(View.VISIBLE);
```

Comment:
So this method is called immediately for the sender device (since the list of files is known at the beginning), and for the receiver it is called after exchange of the metadata.
Only change is that the banner wasn't a part of this method earlier

Reply:
You may end up with 2 different activities (potentially inheriting from an abstract one) depending how divergent the UI is but sure this is good with me